java/main/com/google/privacy/differentialprivacy/LaplaceNoise.java

```diff
@@ -118,6 +118,36 @@ public long addNoise(long x, int l1Sensitivity, double epsilon, @Nullable Double
     return Math.round(addNoise((double) x, l1Sensitivity, epsilon, delta));
   }
 
+  /**
+   * Computes a confidence interval that contains the raw value {@code x} passed to
+   * {@link #addNoise(double, int, double, double, Double)} with a probability greater
+   * or equal to {@code confidenceLevel} based on the specified {@noisedValue} and noise parameters.
+   * Note that {@code delta} must be set to {@code null} because it does not parameterize Laplace
+   * noise. Moreover, {@code epsilon} must be at least 2^-50.
```

Comment:
I  don't think we need to require epsilon > 2^-50 for the confidence intervals. This is a technicallity required for how we compute the noise. But we need to be less carefull with the confidence intervals (since they are not privacy load bearing). So it should be fine to ignore this here.

same for the other method

Reply:
I think we can make a less strict epsilon check too like we have in Go lib. I removed it, Thanks.